### PR TITLE
Removes all relative heresy

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Removes all relative paths, standardises `proc/whatever()` to `/proc/whatever()`

This also enables this checking on dreamchecker so Travis will now beat you if you use relative paths.

This shouldnt affected anything in the game at all. I tested locally and the server still successfully starts and connects to the DB, but I might push for a TM given how much this touches.

## Why It's Good For The Game
Relative paths are very bad and should be fully eradicated. This does that.

## Changelog
:cl: AffectedArc07
fix: Code is now free of relative path heresy
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/law.dm [hunk 1]
```diff
@@ -11,55 +11,57 @@
...
-		if(get_dist(src, usr) > 1 || usr.restrained() || usr.lying || usr.stat || istype(usr, /mob/living/silicon))
-			return
-
-		opened = !opened
-		if(opened)
-			to_chat(usr, "<span class='notice'>The access panel is now open.</span>")
-		else
-			to_chat(usr, "<span class='notice'>The access panel is now closed.</span>")
+// What the fuck even is this
+/obj/machinery/computer/aiupload/verb/AccessInternals()
```
Comment (SteelSlayer) [inline code/game/machinery/computer/law.dm:15]: Uh yeah I agree, what the fuck is this? I've never even seen this verb before. And does this mean you can "open" a computer without a screwdriver?

**Comment by KasparoVy** [comment]
/datum/conflict/war/crusade/DEUS_VULT

File: code/game/machinery/computer/law.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/game/machinery/computer/law.dm:15]: Yeah I saw this while doing this and just cried internally

**Comment by Spacemanspark** [comment]
This is a relatively nice PR

**Comment by AffectedArc07** [comment]
To whoever merges this, tell me to update to master first to avoid any failing builds from newer PRs

**Pull request merged**